client/state/inline-support-article/actions.js

```diff
@@ -24,6 +51,7 @@ export function openSupportArticleDialog( {
 	actionUrl = null,
 	blogId = null,
 } ) {
+	updateUrl( postId );
```

Comment:
Traditional non-thunk action creators are synchronous and don't have any side effects. It's very uncommon to have a side effect in one of them. One way we can get around this is to use middleware. Another, which I'd recommend, would be converting these action creators into async thunks. Side effects are very common and expected in thunks.